subprojects/testng/src/main/java/org/mockito/testng/MockitoBeforeTestNGMethod.java

```diff
@@ -15,7 +15,7 @@
 
 public class MockitoBeforeTestNGMethod {
 
-    private final WeakHashMap<Object, Boolean> initializedInstances = new WeakHashMap<>();
+    private final WeakHashMap<Object, Boolean> initializedInstances = new WeakHashMap<Object, Boolean>();
```

Comment:
Because of that: https://github.com/mockito/mockito/blob/master/build.gradle#L14-L15